packages/terra-worklist-data-grid/CHANGELOG.md

```diff
@@ -1,28 +1,32 @@
 # Changelog
 
 ## Unreleased
-* Breaking Change
-  * Renamed `columnWidth` to `defaultColumnWidth`.
-  
-Added
- * Added additional keyboard navigation in `terra-worklist-data-grid`.
- * Added support for cell selection.
- * Added ability to resize columns via mouse.
- * Added `onColumnResize` prop to notify consumer of a column resize
+
+* Breaking Changes
+  * Removed `columnWidth` prop in favor of the new `defaultColumnWidth` prop.
+  * Renamed `ariaLabelledby` prop to `ariaLabelledBy` to follow proper camelCasing and for consistency with other Terra components.
+
+* Added
+  * Added additional keyboard navigation shortcuts.
+  * Added support for cell selection.
+  * Added ability to resize columns via mouse.
+  * Added `defaultColumnWidth` prop to set the default column width if it is not set per column.
+  * Added `onColumnResize` prop to receive a callback function for notifying consumers of a column resize.
```

Comment:
Do we need to say, "receive a callback function".  The API documentation makes it clear that it is a callback function.

Reply:
I can revert this change.

Reply:
Update: 7ca4211855b34b52740e550e080a43df7877ae3c